apps/desktop/src/lib/file/FileDiff.svelte

```diff
@@ -1,11 +1,18 @@
 <script lang="ts">
+	import { Project } from '$lib/backend/projects';
 	import HunkViewer from '$lib/hunk/HunkViewer.svelte';
 	import LargeDiffMessage from '$lib/shared/LargeDiffMessage.svelte';
+	import { getContext } from '$lib/utils/context';
+	import { getFileExtension } from '$lib/utils/filePath';
+	import { getBinFileSize } from '$lib/utils/fileSize';
 	import { computeAddedRemovedByHunk } from '$lib/utils/metrics';
 	import { getLocalCommits, getLocalAndRemoteCommits } from '$lib/vbranches/contexts';
 	import { getLockText } from '$lib/vbranches/tooltip';
 	import Icon from '@gitbutler/ui/Icon.svelte';
 	import Tooltip from '@gitbutler/ui/Tooltip.svelte';
+	import { join } from '@tauri-apps/api/path';
+	import { convertFileSrc } from '@tauri-apps/api/tauri';
```

Comment:
Thank you so much for this contribution! I had no idea that this `convertFileSrc` API existed, and it does exactly what we need!

Reply:
Thank you for your review! At first, I attempted to access local images using file://, but I encountered the ‘Not allowed to load local resource’ error. After that, I found and referred to https://github.com/tauri-apps/tauri/discussions/1438#discussioncomment-10424231.